### PR TITLE
Added GitHub Action for tests, improved App.js tests

### PR DESCRIPTION
**Pull request by brunogrcsada**
Added a simple GitHub action to run React Jest tests whenever a new commit is pushed!

image

- [x] Added initial tests to App.js
- [x] Fixed unique key issue with list mapping

### CODE REVIEW

File: src/components/volume/volume.test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { render, screen } from '@testing-library/react';
+import Volume from './volume';
+
+it('should...', () => {
```
Comment (flynnn231) [inline src/components/volume/volume.test.js:4]: Add some tests to the Volume component!

**Review by flynnn231 — APPROVED** [review]
Thank you for including the Manual Test Cases file. The action seems to be working well!

**Review by melissalupu — APPROVED** [review]
Wohoo, we have GitHub Actions!! Nicely done

**Review by Jsimpada — APPROVED** [review]
Brilliant Bruno, thank you for working on this

**Pull request merged**